### DISCUSSION TITLE
Regarding Hackathon Sponsorship by Github

### DISCUSSION THREAD

**Discussion by Gustav-Proxi** [community/community#35716] [GitHub Education]
Hey We are planning for a Hackathon in our college and it is our College's first Hackathon, so I was wondering how to get Github sponsorship.

**Comment by jfullstackdev** [comment]
maybe this list ? 

https://github.com/athul/hackathon-sponsor-searches

**Reply by Gustav-Proxi** [reply]
Thank you so much 🤝

**Comment by samus-aran** [comment]
Hi @Vaishakgkumar 👋 

GitHub partners with Major League Hacking, the official student hackathon league.

To organize a hackathon with support from GitHub, simply become an MLH member (it's free!):
[Join the MLH Hackathon Season](https://mlh.io/event-membership)

Fundraising for hackathons is hard. 
Shout out to our organizers making sure all our hacker needs are met throughout events :raised_hands:. 

To help, GitHub is opening grants of up to $1000 USD for [MLH Member Events](https://mlh.io/event-membership) through the end of 2022.
Once approved as an MLH Member Event, you’ll have access to the GitHub Education Grant Application within your MLH application.
[Learn how to apply](https://news.mlh.io/mlh-github-education-grants-supporting-diverse-hacker-communities-03-30-2022)

Please remind your attendees that any student 13+ years of age is welcome to apply individually for GitHub Student Developer Pack membership at: [GitHub Student Developer Pack](https://education.github.com/pack)
The Pack includes valuable offers for real-world development tools provided by GitHub's partners.	The Pack includes valuable offers for real-world development tools provided by GitHub's partners.

**Reply by Gustav-Proxi** [reply]
Thank you 🤗
